src/Tizen.NUI/src/internal/Common/CustomAlgorithmInterface.cs

```diff
@@ -70,7 +70,7 @@ private bool SwigDerivedClassHasMethod(string methodName, global::System.Type[]
                 View currentView = Registry.GetManagedBaseHandleFromNativePtr(current) as View;
                 View proposedView = Registry.GetManagedBaseHandleFromNativePtr(proposed) as View;
 
-                return View.getCPtr(GetNextFocusableView(currentView, proposedView, (View.FocusDirection)direction)).Handle;
+                return View.getCPtr(GetNextFocusableView(currentView, proposedView, (View.FocusDirection)direction, deviceName)).Handle;
```

Comment:
could you check and resolve the warnings shown below?

src/Tizen.NUI/src/internal/Common/CustomAlgorithmInterface.cs#L75
src/Tizen.NUI/src/public/Input/FocusManager.cs#L83
etc ...